clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2610,6 +2609,13 @@ public ListConsumerGroupOffsetsResult listConsumerGroupOffsets(final String grou
             void handleResponse(AbstractResponse abstractResponse) {
                 final FindCoordinatorResponse response = (FindCoordinatorResponse) abstractResponse;
 
+                if (response.retriableError()) {
```

Comment:
nit: a little vexing to see this logic repeated. Maybe we could have a helper?
```java
if (handleFindCoordinatorError(response, future))
  return;
```

Reply:
@hachikuji  Thanks for the review. Updated the PR.